compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/Fir2IrVisitor.kt

```diff
@@ -940,7 +953,9 @@ class Fir2IrVisitor(
                 startOffset, endOffset, tryExpression.typeRef.toIrType(),
                 tryExpression.tryBlock.convertToIrExpressionOrBlock(),
                 tryExpression.catches.map { it.accept(this, data) as IrCatch },
-                tryExpression.finallyBlock?.convertToIrExpressionOrBlock()
+                // Always generate a block for the finally block. When leaving the finally block in the debugger for both
+                // Java and Kotlin there is a step on the end brace. For that to happen we need the block with that line number.
+                tryExpression.finallyBlock?.convertToIrBlock()
```

Comment:
Just to be precise: looks like we should do the same with try and catch, because according to test data, PSI2IR always generates IrBlock for them.

Reply:
Yes, you are right. Made that change for consistency.